lms/templates/api_admin/api_access_request_email_approved.txt

```diff
@@ -0,0 +1,12 @@
+## mako
+Dear ${name},
+
+Your request for access to the ${platform_name} Course Catalog API has been approved. Sign in at ${api_management_url} using the same account you used to request API access to generate your API client credentials.
+
+Notes:
+- ${platform_name} recommends that you bookmark the ${api_management_url} page for easy access.
+- For more information about how to use the API client credential to authenticate your API request, visit the oauth2 documentation at ${authentication_docs_url}.
```

Comment:
Should "credential" be "credentials"?

"visit the oauth2 documentation" --> "see the OAuth 2.0 documentation"